leverage/modules/aws.py

```
@@ -223,3 +237,19 @@ def logout(cli):
+    "--force",
+    is_flag=True,
+    default=False,
+    help="Force refresh all credentials, even if they haven't expired.",
+)
+@pass_container
+def refresh(cli, force):
+    """Refresh credentials for all configured accounts"""
+    try:
+        refresh_all_accounts_credentials(cli, force_refresh=force)
+    except ExitError as e:
+        raise Exit(e.code)
```

Pylint (4.0.1)

[error] Instance of 'ExitError' has no 'code' member

(E1101)